pallets/staking/src/lib.rs

```diff
@@ -2644,6 +2646,32 @@ decl_module! {
                     .map(|p| p.intended_count = new_intended_count)
             })
         }
+
+
+        /// Declare no desire to either validate or nominate.
+        ///
+        /// Effects will be felt at the beginning of the next era.
+        ///
+        /// The dispatch origin for this call must be a GC.
+        /// And, it can be only called when [`EraElectionStatus`] is `Closed`.
+        #[weight = <T as Config>::WeightInfo::chill_from_governance()]
+        pub fn chill_from_governance(origin, identity: IdentityId, stash_keys: Vec<T::AccountId>) {
+            ensure!(Self::era_election_status().is_closed(), Error::<T>::CallNotAllowed);
+            let _controller = ensure_signed(origin)?;
```

Comment:
`ensure_signed` would allow anyone to call this extrinsic.  The staking pallet has 3 types for restricting the `origin`: `RequiredAddOrigin`, `RequiredRemoveOrigin`, and `RequiredCommissionOrigin`.
Here we are forcefully removing an operator, so we want to use the `RequiredRemoveOrigin` restriction:
```suggestion
            T::RequiredRemoveOrigin::ensure_origin(origin)?;
```